test/perf_test/scripts/run.sh

```diff
@@ -34,7 +34,7 @@ popd
 
 export CONFLUENT_HOME="$c_dir/confluent"
 #run test
-mvn clean compile
+mvn clean compile -U
```

Comment:
Out of curiosity: What does -U do?

Reply:
-U is for force downloading the repositories instead of using it from cache. https://stackoverflow.com/questions/43488853/confluent-maven-repository-not-working (Added in PR description as well)